frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.tsx

```diff
@@ -466,6 +489,53 @@ export const WhatsNewMenu = (props: Props) => {
     entries.push(phoneAnnouncement);
   }
 
+  const vpnAndRelayAnnouncement: WhatsNewEntry = {
+    title: l10n.getString("whatsnew-feature-bundle-header"),
+    snippet: l10n.getString("whatsnew-feature-bundle-snippet"),
+    content:
+      props.runtimeData && isBundleAvailableInCountry(props.runtimeData) ? (
+        <WhatsNewContent
+          description={l10n.getString("whatsnew-feature-bundle-body", {
+            monthly_price: getBundlePrice(props.runtimeData),
+            savings: "??%", // Design states 50%
```

Comment:
Still waiting on a confirmation of copy.

Reply:
In case you do get the final percentage before merging this, could you also insert that same value [here](https://github.com/mozilla/fx-private-relay/blob/8e6df3631c2217c0b040c7127d96228c8f1abb71/frontend/src/components/landing/PlanMatrix.tsx#L468) and [here](https://github.com/mozilla/fx-private-relay/blob/8e6df3631c2217c0b040c7127d96228c8f1abb71/frontend/src/components/landing/PlanMatrix.tsx#L298)?

Reply:
Will do. I'll ask Tony if these values are confirmed.